packages/at_client/lib/src/client/at_client_spec.dart

```diff
@@ -572,4 +572,8 @@ abstract class AtClient {
   String? getCurrentAtSign();
 
   EncryptionService? get encryptionService;
+
+  /// Fetches all enrollment requests from the corresponding atServer
+  Future<Map<String, dynamic>> fetchEnrollmentRequests(
+      {String? appName, String? deviceName});
```

Comment:
please explain in doc what happens if we pass appName and deviceName and what is the default behaviou

Reply:
This has been done as part of [17f342a](https://github.com/atsign-foundation/at_client_sdk/pull/1200/commits/17f342af6e4ad3728c14f2a5dc0175f3c60f50d0)